Makefile

```diff
@@ -0,0 +1,23 @@
+
+.PHONY: all
+all:
+	@echo "No default action."
+
+# docker:
+
+docker_subtag := $(shell git describe --tags --dirty)
```

Comment:
I'd prefix this with `dev-` just to make sure it won't replace production build is we run `make docker-push` on a git tag commit, we can still explicitly do this with `make docker-push docker_subtag=vX.Y.Z` but in any case I'd prefer having DockerHub autobuild this or at most a Github Action like we have on `hathor-core`.

My experience with this is that we can have outdated cached layers that could break the build for subtle reasons (this happened a lot with hathor-core). So it's always best to let CI make the production builds, also makes it easier to make multi-arch builds but that's not relevant right now.

Reply:
Done! Thanks. We still need to configure DockerHub Autobuild or Github Action to do it.